CHANGELOG.md

```diff
@@ -7,61 +7,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [10.17.0]
-### Uncategorized
-- Sync `master` with `develop` ([#15040](https://github.com/MetaMask/metamask-extension/pull/15040))
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- Sync `master` with `develop` ([#15013](https://github.com/MetaMask/metamask-extension/pull/15013))
-- README.md: add PASSWORD .metamaskrc instruction ([#15004](https://github.com/MetaMask/metamask-extension/pull/15004))
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- Show how much a cancellation of a smart transaction will cost ([#15011](https://github.com/MetaMask/metamask-extension/pull/15011))
-- Add exclusion for GHSA-pfrx-2q88-qq97, which is in the 3box dependency tree but not our build ([#15005](https://github.com/MetaMask/metamask-extension/pull/15005))
-- fix minor german translation issue. connected=verbunden ([#14997](https://github.com/MetaMask/metamask-extension/pull/14997))
-- Fix 14802 - Prevent children error in page container ([#14809](https://github.com/MetaMask/metamask-extension/pull/14809))
-- Migrate the build script to `yargs` ([#14836](https://github.com/MetaMask/metamask-extension/pull/14836))
-- Fixing tooltip theming by enforcing default theme ([#14992](https://github.com/MetaMask/metamask-extension/pull/14992))
-- Move search icon and add to storybook ([#14940](https://github.com/MetaMask/metamask-extension/pull/14940))
-- Changed accounts fuzzy search threshhold to match all ([#14914](https://github.com/MetaMask/metamask-extension/pull/14914))
-- [FLASK] Fix notifications icon color ([#14941](https://github.com/MetaMask/metamask-extension/pull/14941))
+### Added
+- Add cost estimation for canceling a Smart Transaction on Awaiting Swap page ([#15011](https://github.com/MetaMask/metamask-extension/pull/15011))
+- Add missing subtitles for Secret Recovery Phrase video, including French, Greek, Turkish, Chinese
+ ([#14943](https://github.com/MetaMask/metamask-extension/pull/14943))
+
+### Changed
+- Consolidate all Error Messages in one style component ([#14945](https://github.com/MetaMask/metamask-extension/pull/14945))
+- Improve accessibility by allowing keyboard navigation on Accounts menu ([#14936](https://github.com/MetaMask/metamask-extension/pull/14936))
+- Allow using locally hosted RPCs and Block Explorer Urls when adding an Ethereum Chain programmatically ([#14272](https://github.com/MetaMask/metamask-extension/pull/14272))
 - [FLASK] snaps-skunkworks@0.16.0 ([#14952](https://github.com/MetaMask/metamask-extension/pull/14952))
```

Comment:
```suggestion
- **[FLASK] BREAKING:** Snaps are now required to export `onRpcRequest` to receive RPC requests ([#14952](https://github.com/MetaMask/metamask-extension/pull/14952))
  - All existing snaps will have to be updated to conform to this new standard.
- **[FLASK] BREAKING:** Snap state is now encrypted by default ([#14952](https://github.com/MetaMask/metamask-extension/pull/14952))
  - Breaks existing installed snaps that use `snap_manageState`. All such Snaps must be reinstalled.
- **[FLASK]:** Fix multiple bugs related to snap installation ([#14952](https://github.com/MetaMask/metamask-extension/pull/14952))
```

Categories in order: Changed, Changed, Fixed.

Reply:
@FrederikBolding thank you! I've updated the changes and also included the link with extra info, where I thought it was convenient